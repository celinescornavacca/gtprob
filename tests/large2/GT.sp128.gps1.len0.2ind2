(((((((LC0,((FE0,GE0),MD0)),(AE0,BE0)),(ND0,OD0)),(((((BD0,CD0),RC0),OC0),UA0),((((IC0,JC0),(RD0,SD0)),((HE0,IE0),(WE0,XE0))),MA0))),(((IA0,(WB0,XB0)),((ZA0,AB0),FA0)),((((SC0,TC0),FB0),EB0),(UB0,VB0)))),(((JA0,KA0),((((((JB0,(KB0,LB0)),(FD0,GD0)),(PA0,QA0)),((JD0,KD0),((PC0,QC0),BB0))),((QE0,RE0),(((YD0,ZD0),((SE0,TE0),CE0)),(OB0,PB0)))),(((((YC0,ZC0),XA0),((MB0,NB0),(DC0,EC0))),(OA0,((KC0,(DD0,ED0)),(WC0,XC0)))),(LA0,(((HD0,ID0),AD0),AC0))))),((((((((DE0,EE0),(CB0,DB0)),TA0),(VA0,WA0)),(((TD0,(KE0,LE0)),YA0),HA0)),((LD0,(UD0,VD0)),((BC0,CC0),(ME0,NE0)))),AA0),(BA0,(((((PD0,QD0),HC0),QB0),(MC0,NC0)),(CA0,DA0)))))),(((((GB0,(RB0,(SB0,TB0))),(FC0,GC0)),(((IB0,(UC0,VC0)),((UE0,VE0),JE0)),(HB0,(WD0,XD0)))),((YB0,ZB0),NA0)),(EA0,(((OE0,PE0),(RA0,SA0)),GA0))));
