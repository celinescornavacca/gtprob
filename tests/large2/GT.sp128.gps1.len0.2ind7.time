3.16user 0.10system 0:03.46elapsed 94%CPU (0avgtext+0avgdata 62404maxresident)k
144inputs+0outputs (0major+29435minor)pagefaults 0swaps
