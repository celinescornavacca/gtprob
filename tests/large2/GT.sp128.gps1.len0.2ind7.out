-168.85827129
